portality/scripts/article_cleanup_sync.py

```diff
@@ -0,0 +1,137 @@
+"""
+For each article in the DOAJ index:
+    * Checks that it has a corresponding journal, deletes it otherwise (if -d set)
```

Comment:
So we decided not to put in a -d flag, yes, but you still need to remove the reference to it from the module docstring :)